rust/parser/test/mod.rs

```diff
@@ -1266,73 +1267,70 @@ fn test_parse_empty_string() {
     assert!(parse_query("").is_err());
 }
 
-/*
 #[test]
 fn test_parse_list_one_match() {
-    let queries = r#"match
-$y isa movie;"#;
-
-    let parsed = parse_queries(queries).collect();  // TODO parse_queries -> Iter
-    let expected = vec![try_! {
-    typeql_match!(var("y").isa("movie"))]
-};
-
+    let queries = "match $y isa movie;";
+    let parsed =
+        parse_queries(queries).unwrap().map(|q| q.unwrap().into_match()).collect::<Vec<_>>();
+    let expected = vec![typeql_match!(var("y").isa("movie"))];
     assert_eq!(parsed, expected);
 }
 
 #[test]
 fn test_parse_list_one_insert() {
-    let insertString = "insert\n$x isa movie;";
-    let queries = parse_queries(insertString).collect(toList());
-
-    assertEquals(list(insert(var("x").isa("movie"))), queries);
+    let queries = "insert $x isa movie;";
+    let parsed =
+        parse_queries(queries).unwrap().map(|q| q.unwrap().into_insert()).collect::<Vec<_>>();
+    let expected = vec![typeql_insert!(var("x").isa("movie"))];
+    assert_eq!(parsed, expected);
 }
 
-#[test]
+// #[test]  // TODO antlr4rust bug?
```

Comment:
The behaviour of the Rust version of the generated parser apparently differs from the Java one, in that the Rust version trips over the initial tokens that are in the `HIDDEN` channel instead of skipping them.